python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -822,7 +822,7 @@ def convert_leakyrelu(node, **kwargs):
             inputs=input_nodes,
             outputs=[name],
             name=name)
-    elif act_type in ('gelu'):
+    elif act_type in ('gelu',):
```

Comment:
Why do we add `,` here?

Reply:
otherwise this is not a tuple, and `elu` is in `gelu` so we will never go into the else branch

Reply:
Should we use a list for this instead? Why a tuple?

`elif act_type in ['gelu']:`